agent-install/agent-install.sh

```diff
@@ -2995,7 +2995,16 @@ function check_exch_url_and_cert() {
         cert_flag="--cacert $AGENT_CERT_FILE"
     fi
 
-    local output=$(curl -w %{http_code} -fsS $cert_flag $HZN_EXCHANGE_URL/admin/version 2>/dev/null) || true
+    local TMP_HZN_EXCHANGE_URL=$HZN_EXCHANGE_URL
+    if [[ "$TMP_HZN_EXCHANGE_URL" != "" ]]; then
+	    LAST_CHAR=(${TMP_HZN_EXCHANGE_URL: -1})
+	    # If this URL ends with "/" the curl command will fail... remove the trailing / if it exists
+	    if [[ "$LAST_CHAR" == "/" ]]; then
+		    TMP_HZN_EXCHANGE_URL=${TMP_HZN_EXCHANGE_URL%?}
+	    fi
+    fi
+
+    local output=$(curl -w %{http_code} -fsS $cert_flag $TMP_HZN_EXCHANGE_URL/admin/version 2>/dev/null) || true
```

Comment:
Good catch, can you also fix the same error for ling 2972 and 465?

Reply:
Resolved.. Changed the URL higher up  so other places will not fail